modules/notification/webhook/webhook.go

```diff
@@ -877,10 +877,19 @@ func notifyPackage(sender *user_model.User, pd *packages_model.PackageDescriptor
 		org = nil
 	}
 
+	ctx, _, finished := process.GetManager().AddContext(graceful.GetManager().HammerContext(), fmt.Sprintf("webhook.notifyPackage Package: %s[%d]", pd.Package.Name, pd.Package.ID))
+	defer finished()
+
+	apiPackage, err := convert.ToPackage(ctx, pd, sender)
+	if err != nil {
+		log.Error("Error converting package: %v", err)
+		return
+	}
+
 	if err := webhook_services.PrepareWebhooks(pd.Repository, webhook.HookEventPackage, &api.PackagePayload{
 		Action:       action,
 		Repository:   convert.ToRepo(pd.Repository, perm.AccessModeNone),
-		Package:      convert.ToPackage(pd),
+		Package:      apiPackage,
```

Comment:
Do we want to send possible sensitive information to a webhook? Just like with the Repository one above we just want to send basic information.

Reply:
What do you mean? The `Repository` member a line above could be removed because it is the same as in `Package`.

Reply:
Nevermind my comment, seems like I misunderstood the code.